ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPackageManagementService.scala

```diff
@@ -108,7 +108,9 @@ private[apiserver] final class ApiPackageManagementService private (
           if (request.submissionId.isEmpty)
             SubmissionId.assertFromString(UUID.randomUUID().toString)
           else
-            SubmissionId.assertFromString(request.submissionId)
+            SubmissionId.assertFromString(
```

Comment:
Is it a problem that a user can see a modified `submissionId` (through `ApiConfigManagementService.entries`) in comparison to what he has submitted with?

Reply:
User will not see it through the Ledger API and within the server we have full autonomy.